addons/website_sale/models/res_config_settings.py

```diff
@@ -18,6 +18,7 @@ class ResConfigSettings(models.TransientModel):
     ], string="Shipping Management")
 
     group_delivery_invoice_address = fields.Boolean(string="Shipping Address", implied_group='sale.group_delivery_invoice_address', group='base.group_portal,base.group_user,base.group_public')
+    group_ecom_uom_price = fields.Boolean(string="eCommerce Base Unit Price", implied_group="website_sale.group_ecom_uom_price", group='base.group_portal,base.group_user,base.group_public')
```

Comment:
as discuss this morning, ecom = website_sale

So website_sale.group_website_sale_show_uom_price
or shortly: website_sale.group_show_uom_price 

we know that in the future it will be in sale and not in website_sale so avoid a future rename of the field ;)